Mathlib/Data/Finset/Interval.lean

```diff
@@ -125,4 +128,30 @@ theorem card_Iio_finset : (Iio s).card = 2 ^ s.card - 1 := by
   rw [Iio_eq_ssubsets, ssubsets, card_erase_of_mem (mem_powerset_self _), card_powerset]
 #align finset.card_Iio_finset Finset.card_Iio_finset
 
+lemma covby_insert {i : α} (hi : i ∉ s) : s ⋖ insert i s :=
```

Comment:
It would be good to have this one too:
```suggestion
lemma covby_cons {i : α} (hi : i ∉ s) : s ⋖ cons i s hi :=
```

Reply:
Notably this will need to be outside the scope of the `DecidableEq` argument

Reply:
Though arguably this doesn't belong in this file at all.

Reply:
If you have a better suggestion I'm open to hearing it, but I'm not sure what it should be.

Reply:
(I know I still need to remove the `DecidableEq` dependence for all the `cons` stuff, I'll get to it tomorrow.